File: tests/data/c/comment.c
// { expected = "tabs" }

/*
This comment
    has
        been
    indented
using
    a
        series
    of
spaces.
Nevertheless,
    this
    should
        not
        interfere
    with
    guess-indent.
*/

// And
    // Now
// Let's
    // Do
    // The

// same
    
    // using
        // some
    // inline
// comments.



// ------------------------------------------------------------------------- //

#include <stdio.h>
int main(int argc, char *argv[]) {
	printf("This code has been indented using tabs.\n");
	printf("That's why guess-indent should detect tabs.\n");
	return 0;
}

File: tests/data/lua/comment.lua
-- { expected = nil, disabled = true }

--[[
This is a comment. All the indentation contained inside it should get ignored.
	This line has been indented using a tab.
	This one too.
No indents here...
    Now I used four spaces.
XYZ
	And another tab.
Ok... This should be enought. --]]

--[[
This too is a comment.
	It is different...
It uses a different character sequence to terminate
]]
--

local bar = require("foo")
return bar
